src/AstroImages.jl

```diff
@@ -85,9 +85,21 @@ for n in (8, 16, 32, 64)
     end
 end
 
+mutable struct properties
```

Comment:
Why mutable? It doesn't look like you're going to mutate anything

Reply:
Please see 7th point here: https://docs.google.com/document/d/1-kH0kweDQMw7mHSTCE3TXGTiC-1DQb3lm-04_jHs_VY/edit#heading=h.mdhqag85ullj
The data fields are like brightness, contrast, labels. filters etc. can be specified later by user while working with AstroImage.
Also, it is not necessary that there will be a RGB image. Thus, I could have either added into the same struct with type `Union{nothing, AbstractArray}` or in a mutable struct where all these properties will sit.
I'm not sure if these two are the only way but I asked around slack to reach this conclusion.

Reply:
Also, these operations like brightness, contrast, filter etc. will be applied on `rgb_image`. That's why it is mutable.
We are not changing the original data kept in `data` and `wcs` and we can reset `rgb_image` when we want with `ccd2rgb` function. Users will be able to share their experiments on the visualized `rgb_image`.